File: C/1066.c
#include <stdio.h>
int main()
{
	int c, n, cp = 0,ci = 0,cpo = 0,cn = 0;
	for (c = 0; c < 5; c++)
	{
		scanf("%d",&n);
		if (n % 2 == 0)
		{
			cp += 1;
		}
		if (n % 2 == 1)
		{
			ci += 1;
		}
		if (n > 0)
		{
			cpo += 1;
		}
		if (n < 0)
		{
			cn += 1;
		}
	}
	printf("%d valor(es) par(es)\n",cp);
	printf("%d valor(es) impar(es)\n",ci);
	printf("%d valor(es) positivo(s)\n",cpo);
	printf("%d valor(es) negativo(s)\n",cn);
}

File: python/1066.py
cont = cpar = cimpar = cposi = cnega = 0
while cont < 5:
    value = int(input())
    if value % 2 == 0:
        cpar += 1
    if value % 2 != 0:
        cimpar += 1
    if value > 0:
        cposi += 1
    if value < 0:
        cnega += 1
    cont +=1 
print(f'{cpar} valor(es) par(es)')
print(f'{cimpar} valor(es) impar(es)')
print(f'{cposi} valor(es) positivos(s)')
print(f'{cnega} valor(es) negativo(s)')
